src/3.1/docker-entrypoint.sh

```diff
@@ -184,17 +157,16 @@ done
 
 # Chown the data dir now that (maybe) an initial password has been
 # set (this is a file in the data dir)
-if [[ "$(stat -c %u /data)" = "0" ]]; then
-  chown -R "${user_name}:${group_name}" /data
-fi
+chown -R "${userid}":"${groupid}" /data || :
+chmod -R 700 /data || :
```

Comment:
And here?

Reply:
mounted data volume has an owner which does not match the user the docker container is running as. The directory needs to be writable for the executing user, but we don't have permission to reown or chmod it in that case.